docs/tutorials/connecting-apps/connecting-apps.md

```diff
@@ -0,0 +1,162 @@
+---
+---
+
+{% capture overview %}
+
+In this tutorial you will create a frontend service and a backend
+hello greeter microservice. You will connect the two using a
+Kubernetes Service object.
+
+{% endcapture %}
+
+
+{% capture objectives %}
+
+* Create and run microservices using Deployment objects.
+* Route traffic to the backend using a frontend microservice.
+* Use Service objects to connect the frontend application to the
+  backend application.
+
+{% endcapture %}
+
+
+{% capture prerequisites %}
+
+* {% include task-tutorial-prereqs.md %}
```

Comment:
Same comment here as there, I hope we can have tutorials with `type: LoadBalancer` not require a cloud provider.